packages/uring-trace/uring-trace.0.1.0/opam

```diff
@@ -0,0 +1,53 @@
+opam-version: "2.0"
+synopsis: "IO-uring tracing tool using eBPF probes"
+description: "eBPF-based tracing tool for io-uring. Require's Linux kernel version >= 6.1.0"
+maintainer: ["koonwen@gmail.com"]
+authors: ["Lee Koon Wen"]
+license: ["ISC" "BSD-3-Clause"]
+homepage: "https://github.com/koonwen/uring-trace"
+doc: "https://github.com/koonwen/uring-trace"
+bug-reports: "https://github.com/koonwen/uring-trace/issues"
+depends: [
+  "ppx_deriving" {>= "6.0.2"}
+  "eio" {>= "1.1"}
+  "ocaml"
+  "dune" {>= "3.13"}
+  "dune-site"
+  "cmdliner"
```

Comment:
Noticed in the CI that we are installing

```
-> retrieved cmdliner.1.0.4  (cached)
```

on some platforms.

But you are using the `Cmdliner.Cmd` module which is only available from `1.1.0`, so we need:

```suggestion
  "cmdliner" {>= "1.1.0"}
```

This reflected in the lowerbound errors that turned up after I reran the failing CI jobs (to clear up some transient failures):

```
compilers

    5.1
        uring-trace.0.1.0
            [lower-bounds (failed: Unbound module Cmd)](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/9141711b2bae6019cf673865e320c305bd9e7ab9/variant/compilers,5.1,uring-trace.0.1.0,lower-bounds)
    5.2
        uring-trace.0.1.0
            [lower-bounds (failed: Unbound module "Cmd")](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/9141711b2bae6019cf673865e320c305bd9e7ab9/variant/compilers,5.2,uring-trace.0.1.0,lower-bounds)
```